src/doc/src/guide/continuous-integration.md

```diff
@@ -21,6 +21,42 @@ will not fail your overall build. Please see the [Travis CI Rust
 documentation](https://docs.travis-ci.com/user/languages/rust/) for more
 information.
 
+### GitHub Actions
+
+To test your package on GitHub Actions, here is a sample `.github/workflows/ci.yml` file:
+
+```yaml
+name: Cargo Build & Test
+
+on:
+  push:
```

Comment:
I think this only includes pushes, and no PRs, right?

Reply:
Only on pushes. I could switch to `on: [push, pull_request]` but to be honest, I think the proposed behaviour is the best. If you look at [this PR](https://github.com/SamMorrowDrums/rust-action-test/pull/1), you will see that the checks show up, and if you update the code on a branch, it will trigger the workflow. I think this is closest to a default behaviour that there is.

Reply:
> if you update the code on a branch

This includes a run triggered on the target branch when you merge a PR.

Reply:
The [default configuration](https://github.com/actions/starter-workflows) from the UI of GitHub (click new workflow button and select Rust) is this:

```
name: Rust

on:
  push:
    branches: [ $default-branch ]
  pull_request:
    branches: [ $default-branch ]

env:
  CARGO_TERM_COLOR: always

jobs:
  build:

    runs-on: ubuntu-latest

    steps:
    - uses: actions/checkout@v2
    - name: Build
      run: cargo build --verbose
    - name: Run tests
      run: cargo test --verbose
```

Would you prefer that I use the below config?


```
on:
  push:
  pull_request:
```

The placeholder for $default-branch only works in templates for creating Actions, but not for actual Actions configutations, so I cannot use it without specifiying master/main.

Reply:
I believe that `pull_request` is necessary for CI to run on PRs-from-forks, but otherwise the default template looks pretty reasonable too.